protocol/messenger_communities.go

```diff
@@ -1308,6 +1308,19 @@ func (m *Messenger) CancelRequestToJoinCommunity(request *requests.CancelRequest
 		return nil, err
 	}
 
+	if !community.AcceptRequestToJoinAutomatically() {
```

Comment:
Why do we send cancellation to admins only under this condition?

Reply:
Because if we `AcceptRequestToJoinAutomatically`, that means that admins won't ever see that request to join. It will be handled automatically by the Owner. I think we literally don't send the requests to the Admins in that case as well.
It saves some network bandwidth

Reply:
Makes sense. Thanks